src/SWP/Bundle/CoreBundle/Model/Container.php

```diff
@@ -17,8 +17,24 @@
 use SWP\Bundle\TemplatesSystemBundle\Model\Container as BaseContainer;
 use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
 use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
+use SWP\Component\Revision\RevisionAwareInterface;
+use SWP\Component\Revision\RevisionAwareTrait;
+use SWP\Component\Storage\Model\PersistableInterface;
 
-class Container extends BaseContainer implements TenantAwareInterface
+class Container extends BaseContainer implements TenantAwareInterface, RevisionAwareInterface, PersistableInterface
```

Comment:
there should be `ContainerInterface` created in core bundle and this class would implement it. The new interface should then extend `TenantAwareInterface, RevisionAwareInterface, PersistableInterface`